components/dfs/include/dfs_posix.h

```diff
@@ -16,7 +16,6 @@
 #define __DFS_POSIX_H__
 
 #include <dfs_file.h>
-#include <unistd.h>
```

Comment:
Thanks for your PR. Why do you remove unistd.h? This should not be removed.

Reply:
Got this after handle the first issue.
```
scons: Reading SConscript files ...
scons: done reading SConscript files.
scons: Building targets ...
scons: building associated VariantDir targets: build
CC build/drivers/floppy.o
CC build/kernel/components/dfs/src/dfs_posix.o
/home/teawater/rt-thread/components/dfs/src/dfs_posix.c:113:9: error: conflicting types for 'read'
 ssize_t read(int fd, void *buf, size_t len)
         ^~~~
In file included from /home/teawater/i386-unknown-elf_for_x86_64-pc-linux-gnu/i386-unknown-elf/include/unistd.h:4:0,
                 from /home/teawater/rt-thread/components/dfs/include/dfs_posix.h:19,
                 from /home/teawater/rt-thread/components/dfs/src/dfs_posix.c:13:
/home/teawater/i386-unknown-elf_for_x86_64-pc-linux-gnu/i386-unknown-elf/include/sys/unistd.h:189:25: note: previous declaration of 'read' was here
 _READ_WRITE_RETURN_TYPE read (int __fd, void *__buf, size_t __nbyte);
                         ^~~~
/home/teawater/rt-thread/components/dfs/src/dfs_posix.c:153:9: error: conflicting types for 'write'
 ssize_t write(int fd, const void *buf, size_t len)
         ^~~~~
In file included from /home/teawater/i386-unknown-elf_for_x86_64-pc-linux-gnu/i386-unknown-elf/include/unistd.h:4:0,
                 from /home/teawater/rt-thread/components/dfs/include/dfs_posix.h:19,
                 from /home/teawater/rt-thread/components/dfs/src/dfs_posix.c:13:
/home/teawater/i386-unknown-elf_for_x86_64-pc-linux-gnu/i386-unknown-elf/include/sys/unistd.h:245:25: note: previous declaration of 'write' was here
 _READ_WRITE_RETURN_TYPE write (int __fd, const void *__buf, size_t __nbyte);
                         ^~~~~
scons: *** [build/kernel/components/dfs/src/dfs_posix.o] Error 1
scons: building terminated because of errors.
```

Should I change this part to a ifndef for x86 instead of remove this include?

Reply:
Nop, this is my problem.